bucket/discordchatexporter.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "2.32",
+    "description": "Exports Discord chat logs to a file",
+    "homepage": "https://github.com/Tyrrrz/DiscordChatExporter",
+    "license": "GPL-3.0-only",
+    "url": "https://github.com/Tyrrrz/DiscordChatExporter/releases/download/2.32/DiscordChatExporter.zip",
+    "hash": "3d9543f73a739eb07eaa9db7544c2044dab68232a241fadfdb61f07a2af5df18",
+    "bin": "DiscordChatExporter.exe",
+    "shortcuts": [
+        [
+            "DiscordChatExporter.exe",
+            "Discord Chat Exporter"
+        ]
+    ],
+    "persist": "DiscordChatExporter.exe.config",
```

Comment:
```suggestion
    "persist": "DiscordChatExporter.exe.config",
```

The config file is actually at `Settings.dat`.

`DiscordChatExporter.runtimeconfig.json` only stores some info about prerequisites.

```
<?xml version="1.0" encoding="utf-8"?>
<!-- This configuration file is required by the apphost which runs on legacy .NET Framework for compatibility reasons -->
<configuration>
  <!-- Prefer .NET 3.5 (preinstalled on Windows 7), rollover to .NET 4.x (preinstalled on Windows 8+) -->
  <startup useLegacyV2RuntimeActivationPolicy="true">
    <supportedRuntime version="v2.0.50727" />
    <supportedRuntime version="v4.0" />
  </startup>
</configuration>
```